### ISSUE TITLE
Custom picker uses the wrong timezone to update itself

### ISSUE THREAD

**Issue by mozjag** [DJDavid98/HammerTime#137]
The custom picker should retain the date and time as entered (in the chosen timezone), not use the date and time as they are in the local time zone.

I'm in The Netherlands, and I'm trying to get tags for events in Toronto time. So I set the timezone to America/Toronto, I pick May 31, then set the time to 20:00 and as soon as I enter that "20", the date jumps to June 1, and the "00" brings it to June 2. Alright, slightly annoying, but I'll just click May 31 and all is well. Except it's not. The time shown in the examples is now June 1, 08:00:00 instead of the expected June 1, 02:00:00. If I try to go back in to set it to 20:00, the date jumps again, etc. etc. If I just keep clicking May 31, the example time goes from June 1, 2am to May 31, 8am to May 31, 2pm to May 31, 8pm, round and round.

What appears to be happening is that the picker, on any change, updates its root element with the newly picked date and time using the local timezone instead of the specified timezone.

**Comment by mozjag** [comment]
Almost. The date gets updated correctly, but the time is stuck at midnight (12:00:00 AM), so if I click on the date twice, the example times now show 6:00 AM (instead of the expected 02:00 AM)

**Comment by DJDavid98** [comment]
Yep, I forgot the combined custom input exists, time should be back to working order now

**Comment by mozjag** [comment]
Beautiful, thank you!
